index.js

```diff
@@ -31,6 +31,29 @@ var isPlainObject = function isPlainObject(obj) {
 	return typeof key === 'undefined' || hasOwn.call(obj, key);
 };
 
+// If name is '__proto__', define it as an own property on target
+var setProperty = function setProperty(target, options) {
+	if (options.name === '__proto__') {
```

Comment:
what happens if `Object.defineProperty` is unavailable, like in ES3 engines?

Reply:
I've been thinking about that a bit. It might be good to check for the existence of `Object.defineProperty` and fall back to regular assignment.

This means it would still be possible to modify prototypes in pre-ES5 environments. That seems fine in practice. Some browsers will fall back, but Node won't have to. iojs and Node 0.8 have `Object.defineProperty` available.

(To be frank, Node 0.8 has bigger security concerns anyway)